lib/matplotlib/tests/test_path.py

```diff
@@ -122,6 +122,22 @@ def test_xkcd():
         ax.plot(x, y)
 
 
+@image_comparison(baseline_images=['xkcd_marker'], remove_text=True)
```

Comment:
Could you add `extensions=['png']` to this bit, and only generate the `.png` images? It saves us cluttering up the repository with svg and pdf images.

Reply:
@dstansby, added it